### DISCUSSION TITLE
How do I import another Dashboard JSON into Teslamate

### DISCUSSION THREAD

**Discussion by anupchavda1** [teslamate-org/teslamate#2082] [Q&A]
I have a JSON file that was shared by the community and wanted to add it to my install of Teslamate/Grafana.

How do I import this new dashboard? I followed the basic install instructions and have it all working on my system, but can't see how to add/import new dashboards.

Thanks

**Comment by Chacsam** [comment]
Do you mean basically importing a dashboard in Grafana?
![image](https://user-images.githubusercontent.com/28670803/140746175-4f6abb0f-28b3-4cf5-8747-a855fd329f8c.png)
